src/main/resources/styles/DarkTheme.css

```diff
@@ -0,0 +1,281 @@
+/* Dark theme for PeopleSoft */
+/* Inspired by app by behance user */
```

Comment:
uhhhh this credit is good enough right

Reply:
I've updated it to link to the ui kit and authors